neurobooth_os/stm_session.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, **kwargs):
 
     @staticmethod
     def init_window():
-        if os.getenv("NB_FULLSCREEN") == "false":
+        if config.neurobooth_config.fullscreen == "false":
```

Comment:
This is a Boolean in the updated model, so the string comparison is likely a bug. Should be simplified to
`win = utl.make_win(full_screen=config.neurobooth_config.fullscreen)`

Reply:
fixed